src/BlazorWebView/src/core/MauiAssetFileProvider.Windows.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.Extensions.FileProviders;
+using Microsoft.Extensions.Primitives;
+using Windows.ApplicationModel;
+using Windows.Storage;
+using FileAttributes = System.IO.FileAttributes;
+
+namespace Microsoft.AspNetCore.Components.WebView.Maui
+{
+	public partial class MauiAssetFileProvider
+	{
+		IDirectoryContents PlatformGetDirectoryContents(string subpath)
+			=> new WindowsMauiAssetDirectoryContents(subpath);
+
+		IFileInfo? PlatformGetFileInfo(string subpath)
+		{
+			// TODO: HACK: For now we strip out the folder because it is not preserved in the WinUI assets. We need to address this in the MauiAsset targets.
+			var fileWithoutFolders = Path.GetFileName(subpath);
```

Comment:
@Redth - FYI the current Resizetizer stuff seems to remove folder paths, so if I have a file `wwwroot\css\main\foo.css` the file ends up in `Appx\Assets\foo.css` and I lose the sub-folders. We need a way to preserve the folder structure.

(I tried hacking up some MSBuild goop to do it but couldn't figure it out.)

Do you have any concerns with adding support for sub-folders in MauiAssets?